manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -168,6 +188,16 @@ spec:
         app: vsphere-csi-controller
         role: vsphere-csi
     spec:
+      affinity:
```

Comment:
Comment on line 101 - shouldn't the replica count be 3?

Reply:
No keeping replica count to `1` so customers who do not have multi-master setup can still deploy driver without any issue. Otherwise, deployment will show failures regarding the unavailability of nodes for additional replicas.

Reply:
We can document how customers can add more replicas after deployment of this yaml.

Reply:
Multi master k8s is common. So far I did not see a customer env where they are running k8s with a single master. If multi master is the norm, why not default the replica to 3. For customers who are using single master, we can document that they can run CSI with replica as 1.

Reply:
done. we will need to update pipeline logic to adjust replica to 1 for single master deployment. I will sync up with the team.

Reply:
So, are we recommending that we have one csi-controller pod per control plane node, is it? Until now we were okay with one replica of csi-controller in multi-master setups right!

Reply:
Yes, each master node will have one replica of the csi-controller pod.

We are setting replica count to 3 as default, so if you have more master nodes, replica count needs to be set to match the number of master nodes on the k8s cluster.